xhistogram/test/test_chunking.py

```diff
@@ -0,0 +1,90 @@
+# flake8: noqa
```

Comment:
Is it possible to `# noqa: EXXX` the specific errors, instead of disabling linting globally here? That seems liable to mask actual issues in the future.

Reply:
It was the same error in multiple places, each time the fixtures were passed as arguments, however you can't do `# noqa: EXXX` [apparently](https://stackoverflow.com/a/54454433/3154101), so I tried to silence them all.

If turn the fixtures into functions then I don't think we will have this linting error anyway though.